kis_api.py

```
@@ -1,372 +1,850 @@
+            
+            # 기본 API 테스트 (토큰 유효성 간접 확인)
+            try:
+                test_result = self.broker.fetch_present_balance()
+                if test_result:
+                    self.logger.info("mojito2 클라이언트 초기화 및 토큰 확인 성공")
+                else:
+                    self.logger.warning("mojito2 클라이언트 초기화됨, 토큰 상태 불확실")
+            except Exception as token_test_e:
+                self.logger.warning(f"토큰 테스트 실패 (계속 진행): {token_test_e}")
+            
+            self.logger.info(f"이중 거래소 브로커 초기화 완료 (나스닥 + 뉴욕)")
```

Ruff (0.14.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)